packages/react-auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pangeacyber/react-auth",
-  "version": "0.0.3-beta.1",
+  "version": "0.0.3-beta.2",
```

Comment:
Let's not bump the version yet, there are other bugs i need to fix this weekend.